packaging/packager_rpm.sh

```diff
@@ -5,40 +5,44 @@ readonly distro="${DISTRO:-fedora}"
 readonly distro_version="${VERSION:-30}"
 readonly distro_root="${ROOT:-$HOME}"
 
-# rpm dependencies
-# python3
-dnf -y install python3 python3-coverage
-# python-imaging
+echo "------------- python3 -------------"
+dnf -y install python3 python3-coverage python3-devel
+echo "------------- python-imaging -------------"
 dnf -y install python3-pillow
-# pip dependencies (for dependencies not available as RPM)
+echo "-------------  pip dependencies (for dependencies not available as RPM) -------------"
 dnf -y install gcc libX11-devel libXtst-devel python3-devel libpng-devel python3-pip redhat-rpm-config
+echo "------------- upgrade pip  -------------"
 pip3 install --upgrade pip
-# contour, template, feature, cascade, text matching
+
+echo "------------- contour, template, feature, cascade, text matching -------------"
 dnf -y install python3-numpy python3-opencv
-# text matching
 dnf -y install tesseract tesseract-devel
 dnf -y install gcc-c++
-pip3 install pytesseract==0.3.4 tesserocr==2.5.1
-# deep learning
-pip3 install torch==1.8.1 torchvision==0.9.1
-# screen controlling
-if (( distro_version <= 32 )); then
-    pip3 install autopy==4.0.0
```

Comment:
The section above has lost useful information in separate which dependencies are for all but text matching and which are specifically for text matching.

Reply:
So let's just separate text matching for identical sections like in the case of the debian packager.